stdlib/internals.ncl

```diff
@@ -38,6 +38,11 @@
           %seal% sealing_key (%chng_pol% label) value,
 
   "$forall" = fun sealing_key polarity contract label value =>
+    let flip = match {
+      `Positive => `Negative,
+      `Negative => `Positive,
+    } in
```

Comment:
It's usually a detail, but for low-level internals, this should be slightly more performant, as you only allocate `flip` once and for all, instead of doing it at each function call (it's called let-floating optimization in Haskell, but we don't do any kind of optimizations yet in Nickel :upside_down_face: )

```suggestion
  "$forall" =
    let flip = match {
      `Positive => `Negative,
      `Negative => `Positive,
    } in
    fun sealing_key polarity contract label value =>
```